.travis.yml

```diff
@@ -36,6 +32,7 @@ install:
 
 before_script:
 - export -n _JAVA_OPTIONS # https://github.com/travis-ci/travis-ci/issues/8408
+- source environment
```

Comment:
What's the reason for adding this here? It's already part of `before_install`. One of the two invocations is redundant, correct?

Reply:
The reason is in the PR description.